src/main/resources/jenkins/plugins/openstack/compute/slaveopts/BootSource/Image/config.jelly

```diff
@@ -3,4 +3,10 @@
     <f:entry title="Name" field="name">
         <f:select/>
     </f:entry>
+    <f:entry title="Create New Volume" field="createNewVolume">
+        <f:booleanRadio />
+    </f:entry>
+    <f:entry title="Volume Size (GB)" field="volumeSize">
+        <f:number/>
+    </f:entry>
```

Comment:
This would better be done by https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/optionalBlock.jelly or something similar to better indicate they are related (the size is pointless without the checkbox/radio selected). Also, It can then be represented by `Integer` with `null` indicating the volume should not be created.

Reply:
I will look into this, I didn’t know about that feature. I’ll address it in a follow up commit shortly.